portfolio-alex.css

```diff
@@ -114,12 +154,59 @@ a{
 	width:100%;
 	background-color:black;
 }
+@media only screen and (max-width: 900px) {
+  .project1{
+	  width:50%;
+	  height:285;
+  }
+}  
+@media only screen and (max-width: 900px) {
+  .project2{
+	  width:50%;
+	  height:285;
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .project3{
+	  width:50%;
+	  height:285;
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .project4{
+	  width:50%;
+	  height:285;
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .project5{
+	  width:50%;
+	  height:285;
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .project6{
+	  width:50%;
+	  height:285;
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .project7{
+	  width:50%;
+	  height:285;
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .project8{
+	  width:50%;
+	  height:285;
+  }
+} 
```

Comment:
All of this does the same, why not have one class (i.e. `.project`) and apply the styling to that.